contrib/platform/src/com/sun/jna/platform/mac/CoreFoundationUtil.java

```diff
@@ -0,0 +1,142 @@
+/*
+ * Copyright (c) 2019 Daniel Widdis
+ *
+ * The contents of this file is dual-licensed under 2
+ * alternative Open Source/Free licenses: LGPL 2.1 or later and
+ * Apache License 2.0. (starting with JNA version 4.0.0).
+ *
+ * You can freely decide which license you want to apply to
+ * the project.
+ *
+ * You may obtain a copy of the LGPL License at:
+ *
+ * http://www.gnu.org/licenses/licenses.html
+ *
+ * A copy is also included in the downloadable source code package
+ * containing JNA, in file "LGPL2.1".
+ *
+ * You may obtain a copy of the Apache License at:
+ *
+ * http://www.apache.org/licenses/
+ *
+ * A copy is also included in the downloadable source code package
+ * containing JNA, in file "AL2.0".
+ */
+package com.sun.jna.platform.mac;
+
+import static com.sun.jna.platform.mac.CoreFoundation.kCFStringEncodingUTF8;
+
+import java.util.Collection;
+
+import com.sun.jna.Memory;
+import com.sun.jna.platform.mac.CoreFoundation.CFAllocatorRef;
+import com.sun.jna.platform.mac.CoreFoundation.CFNumberType;
+import com.sun.jna.platform.mac.CoreFoundation.CFTypeRef;
+import com.sun.jna.ptr.IntByReference;
+import com.sun.jna.ptr.LongByReference;
+
+/**
+ * Provides utilities for Core Foundations
+ */
+public class CoreFoundationUtil {
+
+    private static final CoreFoundation CF = CoreFoundation.INSTANCE;
+
+    public static final CFAllocatorRef ALLOCATOR = CF.CFAllocatorGetDefault();
```

Comment:
I have a bad feeling here - from the documentation (https://developer.apple.com/documentation/corefoundation/1521325-cfallocatorgetdefault?language=objc):

> A reference to the default allocator for the current thread. **If none has been explicitly set**, 

So the value of `CFAllocatorGetDefault` is thread and time dependend. Buffering it sounds like a bad idea.